README.md

```diff
@@ -24,6 +24,29 @@ In order to work, KeyCastr must be included in the list of applications with Acc
 
 To add KeyCastr to the list click the <kbd>&plus;</kbd> button and select KeyCastr from the file system.
 
+> NOTE: If KeyCastr is already in the list, then click the <kbd>&minus;</kbd> button and add KeyCastr again to be certain that the right application is chosen.
+
+## Input Monitoring
+
+Also, KeyCastr needs your explicit permission to to monitor inputs. You can set this in the Input Monitoring section under the Security & Privacy system preferences.
```

Comment:
Hmm, I'm not sure this is actually true. I haven't needed to do this on my machine and I'm on macOS 10.15.3

Reply:
This was needed in a clean installation of KeyCastr on macOS 10.15 Catalina, however in that case it replaces the need for Accessibility permissions. So this instruction could use further refinement.

Reply:
We could change the wording to “KeyCastr _may_ need your explicit permission on certain versions of macOS”